chef/cookbooks/barclamp/libraries/barclamp_library.rb

```diff
@@ -203,6 +215,31 @@ def usage
           self.owner
         end
 
+        def multipath?
+          self.class.multipath?(@device)
+        end
+
+        def held_by_multipath?
+          # we need to check if the holders of a device (if it has holders)
+          # are a multipath, for example:
+          # 0QEMU_QEMU_HARDDISK_00002 dm-1 QEMU,QEMU HARDDISK
+          # size=10G features='0' hwhandler='0' wp=rw
+          # -+- policy='service-time 0' prio=1 status=active
+          # - 0:0:0:2 sdc 8:32   active ready running
+          # -+- policy='service-time 0' prio=1 status=enabled
+          # - 0:0:0:3 sdb 8:16   active ready running
```

Comment:
Good point, I thought I had the command in the first line but seems like I skipped it. Adding it back!